server.go

```diff
@@ -732,12 +748,12 @@ func tokenHandler(w http.ResponseWriter, r *http.Request) {
 		//}
 		log.Printf("forwarding request to /register for part 2")
 		w.Header().Set("Location", "/register")
-		w.WriteHeader(301)
+		w.WriteHeader(http.StatusTemporaryRedirect)
```

Comment:
not sure if you or I did this (prolly me). It is idiomatic to use constants over ints.